watcher/src/watcher_db.rs

```diff
@@ -188,7 +236,7 @@ impl WatcherDB {
         Ok(())
     }
 
-    /// Get the highest block that all urls have synced.
+    /// Get the highest block that all currently-configured urls have synced.
```

Comment:
@eranrund - I think you're right that calling this "highest_synced_block" is confusing (you raised this early on). Would now be the time to change this to something like "highest_common_block?"